README.md

```diff
@@ -80,6 +80,50 @@ case object GetPeople extends CollectionQuery[Seq, Person] {
 val person = db(GetPeople).head // Person(Coda Hale,chale@example.com,29)
 ```
 
+You can also use the sql string interpolator:
```

Comment:
Very minor, but I'd like if we wrote "SQL" instead of "sql" for consistency with our other documentation.